CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 Changes
 =======
 
+Beta 0.4.3
+----------
+Developer changes:
+* Added method `get_metadata(self, sample_id, project_id)` to allow other metadata to be fetched from the `api` module
+* Added method `send_metadata(self, metadata, project_id, sample_id)` to allow other metadata to be sent using the `api` module
+* Added model `OtherMetadata` to support `send_metadata` method
```

Comment:
Ya sounds good! I had it as OtherMetadata as that is the location it is stored in IRIDA so that there wouldn't be any confusion with what metadata means